microsoft-edge/toc.yml

```diff
@@ -1307,6 +1302,21 @@
               href: webview2/concepts/working-with-local-content.md
               displayName: 
 
+# -----------------------------------------------------------------------------
+        - name: Test and automate WebView2 apps
```

Comment:
Can we change the title to "Test and automation in WebView2 apps"?

Reply:
the new wording fits with other wv2 TOC buckets, which aren't a verb.
https://learn.microsoft.com/microsoft-edge/webview2/